python/pyspark/context.py

```diff
@@ -230,6 +230,14 @@ def _do_init(self, master, appName, sparkHome, pyFiles, environment, batchSize,
         self.pythonExec = os.environ.get("PYSPARK_PYTHON", 'python3')
         self.pythonVer = "%d.%d" % sys.version_info[:2]
 
+        if self.pythonVer < (3, 7):
+            with warnings.catch_warnings():
+                warnings.simplefilter("once")
+                warnings.warn(
+                    "Python 3.6 support will be deprecated in Spark 3.2.",
```

Comment:
`Python 3.6 support is deprecated in Spark 3.2.`?

Reply:
Cool.